### PR TITLE
Documentation & deploy

### PR DESCRIPTION
**Pull request by c-castillo**

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Pinning server deployment
+
+The pinning server is a node.js process that is running using a the PM2 process manager.
+
+## Deployment
+
+Server deployment is manual at the moment. Continuous deployment is gonna be added in the future.
+
+To deploy it you should enter `ssh` into the machine, and then `cd` into `3box-pinning-server`
```
Comment (oed) [inline DEPLOYMENT.md]: Don't you also need to do a git pull in that dir?

File: .env.default [hunk 2]
```diff
@@ -0,0 +1 @@
+IPFS_PATH=/opt/ipfs
```
Comment (oed) [inline .env.default]: Is this used by the ipfs node? Don't you need to pass the options to the constructor?

File: DEPLOYMENT.md (hunk 1 above)
Comment (c-castillo) [reply DEPLOYMENT.md]: The script handles that

File: .env.default (hunk 2 above)
Comment (c-castillo) [reply .env.default]: Yes, you're right. Forgot to add the constructor to use the env variable

File: DEPLOYMENT.md (hunk 1 above)
Comment (oed) [reply DEPLOYMENT.md]: ok, cool.

**Comment by c-castillo** [comment]
@oed added what was missing

**Pull request merged**